pkg/providers/aws/cluster_network_provider.go

```diff
@@ -899,13 +899,13 @@ func (n *NetworkProvider) reconcileStandaloneRouteTableTags(ctx context.Context,
 					{
 						Key:   aws.String(tagDisplayName),
 						Value: aws.String(DefaultRHMIVpcRouteTableNameTagValue),
-					}, croOwnerTag,
+					}, genericToEc2Tag(croOwnerTag),
 				},
 			})
 			if err != nil {
 				return errorUtil.Wrap(err, "unable to tag route table")
 			}
-			logger.Infof("successfully tagged route table: %s for clusterID: %s", aws.StringValue(routeTable.RouteTableId), aws.StringValue(croOwnerTag.Value))
+			logger.Infof("successfully tagged route table: %s for clusterID: %s", aws.StringValue(routeTable.RouteTableId), aws.String(croOwnerTag.value))
```

Comment:
I think this should be the `StringValue`

Reply:
i'll remove the aws.String bit as we discussed.